client/my-sites/site-settings/form-analytics.jsx

```diff
@@ -141,6 +141,8 @@ export class GoogleAnalyticsForm extends Component {
 							type: TYPE_PREMIUM,
 							...( siteIsJetpack ? { term: TERM_ANNUALLY } : {} ),
 						} ) }
+						jetpack={ siteIsJetpack }
```

Comment:
I think this should be `isJetpack`, right?

https://github.com/Automattic/wp-calypso/blob/73d0d63f21ca667747a7f97c534daea652bbdc0a/client/blocks/upsell-nudge/index.jsx#L41

```suggestion
						isJetpack={ siteIsJetpack }
```